tests/e2e/e2e_egress_validator_test.go

```diff
@@ -45,6 +48,18 @@ var _ = OSMDescribe("Test Submit Egress Policy",
 				egressOut, err := palpha1.PolicyV1alpha1().Egresses(Td.OsmNamespace).Create(ctx, egressIn, v1.CreateOptions{})
 				Expect(err).ShouldNot(HaveOccurred())
 				Expect(egressOut).ShouldNot(BeNil())
+				//now test spec Matches. Should fail
+				derr = palpha1.PolicyV1alpha1().Egresses(Td.OsmNamespace).Delete(ctx, invalidTestName, v1.DeleteOptions{})
+				if derr != nil {
+					//don't error because name is not there first test but report
+					GinkgoWriter.Write([]byte("Error deleting last test service:\n" + derr.Error()))
+				}
+				egressIn.SetName(invalidTestName)
+				egressIn.Spec.Sources = []policy.SourceSpec{{Kind: validTestKind, Name: invalidTestName, Namespace: validTestName}}
+				egressIn.Spec.Matches = []corev1.TypedLocalObjectReference{{Kind: invalidTestKind}}
```

Comment:
I think this will fail schema validation as well (i'm guessing it currently does?), since it is required in the schema

Reply:
hmmm. we may not b saying same thing but no.... this is designed to fail on Matches only.

Reply:
does this currently pass?